README.md

```diff
@@ -83,11 +87,16 @@ python tuning/sft_trainer.py  \
 ```
 
 ### Multiple GPUs with FSDP
+
 ```bash
-torchrun \
---nnodes=1 \
---nproc_per_node=8 \ 
---master_port=1234 \
+MODEL_PATH=llama-7b-hf # Huggingface model id or path to a checkpoint
+DATA_PATH=twitter_complaints.json # Path to the dataset
+OUTPUT_PATH=out # Path to the output folder where the checkpoints are saved
+MASTER_PORT=1234 # The port at which the process with rank 0 listens to
+MASTER_ADDR=x.x.x.x # The IP addresss of the node with rank 0
+
+accelerate launch --main_process_ip $MASTER_ADDR --main_process_port $MASTER_PORT \
+--config_file config/accelerate_fsdp_llama_2_procs.yaml \
```

Comment:
does this change also need to be added? https://github.com/foundation-model-stack/fms-hf-tuning/pull/91/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R105

Reply:
The previous `--bf16` would fire up mixed precision training whereas `--torch_dtype` would load the model weights in the given dtype. We can use both in conjunction. However, I have moved on with Fabian's change as its much more simple without bringing in mixed precision training (using --bf16) and loading the model in dtype bfloat16 which is typical (even I have used this in my command as shown in the PR description (https://github.com/foundation-model-stack/fms-hf-tuning/pull/92#issue-2183504285)) This is part of the cherry picked commit and credited him :)